### PR TITLE
JsonNet: custom functions and aliases

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/KAC-12

Changes:
- Added the ability to define functions and aliases for JsonNet.

### CODE REVIEW

File: internal/pkg/jsonnet/context_test.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+	expected := `
+{
+  "alias1": "aliasValue",
+  "alias2": 3,
+  "func1": "---param1---param2---",
+  "func2": "***param1***param2***",
+  "func2Alias": "***param1***param2***",
+  "var1": "value1",
+  "var2": "value2"
+}
```
Comment (webrouse) [inline internal/pkg/jsonnet/context_test.go:71]: Pozri prosim najprv tento test, z toho by to malo byt jasne.

Narazil som na to, ze budeme potrebovat definovat aj vlastne JsonNet funkcie a aliases.
Tak som nahradil `jsonnet.VariablesValues` -> `jsonnet.Context`.

Vsetko je to otestovane v tomto test:
- ext variable
- native (Go) function
- native (Go) function + alias (globalna premenna)
- alias (globalna premenna)

File: internal/pkg/jsonnet/jsonnet.go [hunk 2]
```diff
@@ -115,29 +114,3 @@ func DefaultOptions() formatter.Options {
 		CommentStyle:     formatter.CommentStyleSlash,
 	}
 }
-
-func registerVariables(vm *jsonnet.VM, vars VariablesValues) {
-	for k, v := range vars {
-		if v == nil {
-			vm.ExtNode(k, &ast.LiteralNull{})
```
Comment (webrouse) [inline internal/pkg/jsonnet/jsonnet.go:122]: Presunute do `context.go`.

File: internal/pkg/jsonnet/jsonnet_test.go [hunk 3]
```diff
@@ -15,39 +15,6 @@ func TestEvaluate(t *testing.T) {
 	assert.Equal(t, "{\n  \"foo\": \"bar\"\n}\n", json)
 }
 
-func TestEvaluate_Variables(t *testing.T) {
-	t.Parallel()
-
-	code := `{
```
Comment (webrouse) [inline internal/pkg/jsonnet/jsonnet_test.go:21]: Presunute do `context_test.go`.

File: internal/pkg/jsonnet/context_test.go (hunk 1 above)
Comment (webrouse) [reply internal/pkg/jsonnet/context_test.go:71]: Note: ako vidis, JsonNet kniznica zoradzuje kluce podla abecedy, zatial neriesime.

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**